controllers/noobaa-rules.yaml

```diff
@@ -59,16 +61,14 @@
            system_type: NooBaa
            system_vendor: Red Hat
            origin: data-federation-service
-         record: noobaa_system_health_status
+         record: namespace:noobaa_system_health_status
    - name: bucket-state-alert.rules
      rules:
-       - alert: BucketPolicyErrorState
+       - alert: NooBaaBucketPolicyErrorState
          annotations:
            description: A Bucket Policy {{ $labels.bucket_name }} is in error
              state for more than 5m
-           message: A Bucket Policy Is In Error State
-           severity_level: warning
-           storage_type: NooBaa
+           summary: A Bucket Policy Is In Error State
          expr: |
            NooBaa_namespace_bucket_status{bucket_name=~".*"} == 0
```

Comment:
This would need to be wrapped with an *_over_time() function to avoid alert resets on failed scrapes. The `bucket_name=~".*"` label matcher is also not needed.

```suggestion
           avg_over_time(NooBaa_namespace_bucket_status[5m]) < 0.8
```

https://www.robustperception.io/alerting-on-gauges-in-prometheus-2-0/

Reply:
Just a doubt, I looked at the aforementioned link, and towards the end it says,

![image](https://user-images.githubusercontent.com/33557095/181062728-d68156fc-fdc2-4e9f-a509-c7937d7f88ab.png)

But isn't this usecase already accomplished by `for`?

Reply:
no, the reason for using avg_over_time() (or another _over_time function) is to avoid that a single failed scrape resolves the alert. When your expression is `NooBaa_namespace_bucket_status == 0`, you could have the following timeline:

* t, NooBaa_namespace_bucket_status == 0 -> alert pending
* t+1m, NooBaa_namespace_bucket_status == 0 -> alert pending
* ...
* t+5m, NooBaa_namespace_bucket_status == 0 -> alert firing (because the `for` duration has elapsed)
* t+6m, failed scrape -> alert resolved (because NooBaa_namespace_bucket_status == 0 returns no data)
* t+7m, NooBaa_namespace_bucket_status == 0 -> alert pending
* ...

Reply:
ACK. But why is it 0.8 (in your suggestion above)?